src/components/Timeline/updates/TimelineJourneyStart.tsx

```diff
@@ -0,0 +1,34 @@
+import { FormattedMessage } from 'react-intl';
+import { Typography } from '@material-ui/core';
+
+import UpdateContainer from './elements/UpdateContainer';
+import ZetkinJourneyInstanceCard from 'components/ZetkinJourneyInstanceCard';
+import ZetkinPersonLink from 'components/ZetkinPersonLink';
+import { ZetkinUpdateJourneyStart } from 'types/updates';
+
+interface TimelineJourneyStartProps {
+  update: ZetkinUpdateJourneyStart;
+}
+
+const TimelineJourneyStart: React.FC<TimelineJourneyStartProps> = ({
+  update,
+}) => {
+  return (
+    <UpdateContainer
```

Comment:
The styling doesn't match the designs - not counting this as a blocker because it's functional and deadline approaching...